### ISSUE TITLE
[Bug]: BETA - Building a Registration Form - Step 31

### ISSUE THREAD

**Issue by jeremyfiel** [freeCodeCamp/freeCodeCamp#44634]
## Affected page

[Learn HTML Forms By Building A Registration Form](https://www.freecodecamp.org/learn/2022/responsive-web-design/learn-html-forms-by-building-a-registration-form/step-31)

Challenge indicates adding options to the dropdown menu.  One of them is `freeCodeCamp YouTube` but the test is looking for `freeCodeCamp YouTube Channel`

This would be a #goodfirstissue for someone to contribute. 

I don't need help with this.  I'm only filing it to give someone a chance to work on it. 

Happy New Year!

## Your code

```js
      <fieldset>
        <label>Upload a profile picture: <input type="file" /></label>
        <label>Input your age (years): <input type="number" min="13" max="120" />
			  </label>
        <label>How did you hear about us?
          <select>
            <option>(select one)</option>
            <option>freeCodeCamp News</option>
            <option>freeCodeCamp YouTube</option>
            <option>freeCodeCamp Forum</option>
            <option>Other</option>
          </select>
        </label>
      </fieldset>



```

## Expected behavior

Tests pass successfully

## Screenshots

![image](https://user-images.githubusercontent.com/32110157/147840479-e965d3a6-f365-451c-88a4-ef6a1a42d2ac.png)

## System

- Device: [e.g. iPhone6, Laptop]
- OS: [e.g. iOS 14, Windows 10, Ubuntu 20.4]
- Browser: [e.g. chrome, safari]
- Version: [e.g. 22]

**Comment by xoxoKaralee** [comment]
FYI i believe this is the same as closed issue [#44602](https://github.com/freeCodeCamp/freeCodeCamp/issues/44602)

**Comment by jeremyfiel** [comment]
Looks like it.   Has this been deployed yet?

**Comment by xoxoKaralee** [comment]
> Looks like it. Has this been deployed yet?

No idea. I just have my own unrelated issue and am working through submitting a PR for it, and had remembered seeing this issue so i went to go find it. It looks like it was addressed and merged in PR #44607 , but I'm completely new to this whole process, so i'm still learning and not sure why it would still show as not fixed.

**Comment by gikf** [comment]
Thank you for reporting this issue.

This is a standard message notifying you that the problem you mentioned here is present in production, but that it has already been fixed in `main`. This means that the next time code is pushed to production, this problem should be fixed. Because of this, we're closing this issue.

If you think we're wrong in closing this issue, please request for it to be reopened and add further clarification. Thank you and happy coding.
